completions/apt-get

```diff
@@ -1,20 +1,22 @@
 # Debian apt-get(8) completion                             -*- shell-script -*-
 
 # @since 2.12
-# TODO:API: make it a generator
-_comp_xfunc_apt_get_installed_packages()
+_comp_xfunc_apt_get_compgen_installed_packages()
 {
     if [[ -f /etc/debian_version ]]; then
         # Debian system
-        _comp_xfunc dpkg installed_packages
+        _comp_compgen -x dpkg installed_packages
     else
         # assume RPM based
-        _comp_xfunc rpm installed_packages
+        _comp_compgen -x rpm installed_packages
     fi
 }
 
-_comp_deprecate_func 2.12 _comp_cmd_apt_get_installed_packages \
-    _comp_xfunc_apt_get_installed_packages
+# @deprecated 2.12
+_comp_cmd_apt_get_installed_packages()
+{
+    _comp_compgen -ax apt-get installed_packages "$@"
+}
```

Comment:
I think this did not exist before 2.12, so perhaps we can remove the function altogether?
```suggestion
```

Reply:
Thank you. OK, it was introduced as a function `_apt_get_installed_packages` in commit ef1b97befb02a4f710c498a37df5c0a8f29b8f5b on 2020-11-07 (which is later than 2.11 release on 2020-07-25).